packages/eslint-plugin-udemy/rules/no-action-bound/README.md

```diff
@@ -0,0 +1,29 @@
+# No @action.bound
+
+`udemy/no-action-bound` rule checks for any uses of `@action.bound` and
+provides a fix using two separate decorators: `@autobind @action`.
+
+This rule prevents the use of `@action.bound` in favor of of the more explicit
+`@autobind @action` to maintain consistency and clarity.
+
+## Rule details
+
+This rule takes no arguments.
+
+The following patterns are considered warnings:
+
+```js
+
+@action.bound myMethod() {};
+
+```
+
+The following patterns are not considered warnings:
```

Comment:
We generally don't use warnings because everyone always ignores them. Everything is either an error or allowed.